src/common/utility.h

```diff
@@ -113,6 +113,10 @@ char *dt_copy_filename_extension(const char *filename1, const char *filename2);
 // replaces all occurences of a substring in a string
 gchar *dt_str_replace(const char *string, const char *search, const char *replace);
 
+// call class function to add or remove CSS classes
```

Comment:
those routines should be moved into gtk.h & gtk.c and renamed `dt_gui_add_class()` and `dt_gui_remove_class()`.

Reply:
Ok, will do that

Reply:
done